### PR TITLE
New Crowdin updates

### PR DESCRIPTION
**Pull request by merc1er**

### CODE REVIEW

**Comment by ftrader** [comment]
@merc1er should I just go ahead and resolve these?

- lang/fr/index.yaml
- lang/fr/navbar.yaml 

Last time I looked at this conflict it seemed simple to resolve.

**Comment by merc1er** [comment]
> @merc1er should I just go ahead and resolve these?
> 
> 
> 
> - lang/fr/index.yaml
> 
> - lang/fr/navbar.yaml 
> 
> 
> 
> Last time I looked at this conflict it seemed simple to resolve.

Ok! Thank you.  
Make sure NOT to merge the Russian files though.

**Comment by ftrader** [comment]
> Make sure NOT to merge the Russian files though.

Why are they included in the changeset then?

**Comment by merc1er** [comment]
This is automatically created by Crowdin (even though it shows my name).
I can remove them easily from Crowdin directly

**Comment by ftrader** [comment]
> I can remove them easily from Crowdin directly

If we want to hide them because they are just automatic / not up to good quality or not reviewed yet, then maybe yeah, doing it directly in Crowdin to not make them appear in PRs for now...

File: lang/zh/index.yaml [hunk 1]
```diff
@@ -14,26 +14,26 @@ donate:
...
-    goal2: 在共识变更的课题上，实现基于证据与合作的决策过程。
-    goal3: 保持和扩大BCH的网络效应。
-    goal4: 通过性能改进和更先进的网络传输协议，在实现全球规模的扩容上取得进展。
-    goal5: 在制定规格与测试计划上与其他团队合作，以使节点多样化，促成更广泛的参与，并保护BCH网络免于重大的灾害。
+    title: Medium term
+    goal1: Demonstrate sustainable client development based on wide ecosystem input and a transparent, voluntary funding model.
+    goal2: Demonstrate evidence-based, collaborative decision making on consensus changes.
+    goal3: Maintain and expand network effect where possible.
+    goal4: Make progress on the scaling work for global adoption through both processing and network performance improvements.
+    goal5: Cooperate on specification and testing initiatives to diversify the node landscape, stimulate broader participation and protect the BCH network from catastrophic failures.
```
Comment (merc1er) [inline lang/zh/index.yaml:33]: Needs to be manually reset to the original file

**Pull request merged**